git/README.md

```diff
@@ -11,12 +11,3 @@
 GIT Cheatsheet:
 https://education.github.com/git-cheat-sheet-education.pdf
 https://www.freecodecamp.org/news/10-important-git-commands-that-every-developer-should-know/
-
```

Comment:
do not delete these! : )

Reply:
ahem sorry about that - they looked like erroneous copy/paste material from a different section (they relate to terminal not git)